android/PhysicalWeb/app/src/main/AndroidManifest.xml

```diff
@@ -89,7 +89,7 @@
 
         <activity
             android:name=".OobActivity"
-            android:label="@string/title_activity_oob"
+            android:label="OobActivity"
```

Comment:
Ah, I see...you could have kept it if it's config to config... no worries though.  Just leave it.